.github/actions/setup_rust_cargo/action.yaml

```diff
@@ -9,6 +9,16 @@ runs:
       with:
         targets: wasm32-unknown-unknown
 
+    - name: Install protoc
+      shell: bash
+      run: |
+        if [ -f /opt/homebrew/bin/brew ]; then
+          /opt/homebrew/bin/brew install protobuf
+        elif [ -f /usr/bin/apt ]; then
+          sudo /usr/bin/apt update
+          sudo /usr/bin/apt install -y protobuf-compiler
+        fi
+
```

Comment:
The arduino setup action was relentless with rate limiting (not sure why they don't allow bypassing the API call), and I didn't want to pass our whole GH token rn https://github.com/marketplace/actions/setup-protoc